bin/next-init

```diff
@@ -12,14 +12,18 @@ const argv = parseArgs(process.argv.slice(2), {
 
 const dir = resolve(argv._[0] || '.')
 
-exists(join(dir, 'package.json'))
+exists(dir)
 .then(async present => {
+  if (!present) {
+    await mkdir(dir)
```

Comment:
Picked `mkdirp-then` out of many alternatives. Same author & similar dependencies to `mz`.

Now `next init a/b/c` works.

https://github.com/zeit/next.js/pull/136/commits/3402f7c2107523a3332996f6aff3ddbb243aaba1